server/run.sh

```diff
@@ -12,15 +12,19 @@ if [ ! -f "config.py" ]; then
   cp config.example.py config.py
 fi
 
+# delete to avoid some module import problems
+if [ -f "../optima/optima.pyc" ]; then
+  rm ../optima/optima.pyc
```

Comment:
I put this in line because in one of the recent pulls, I got the problem again. It's a real head-scratcher because the error gives no indication of the problem. By having it in the script, I hope it will eliminate the problem for other people as well. My suggestion is to leave it in for a while, and then take it out when most people's systems are clear. 

Also, the idea of removing all .pyc is an excellent one because I remember many bugs in the past related to invalidated caches.